sway-lsp/src/core/session.rs

```diff
@@ -38,7 +40,7 @@ pub struct CompiledProgram {
 #[derive(Debug)]
 pub struct Session {
     pub documents: Documents,
-    pub config: RwLock<SwayConfig>,
+    pub config: RwLock<Config>,
```

Comment:
Let's remove config from the Session struct and just initialize swayfmt where it's used.